sp800-63a/sec5_proofing.md

```diff
@@ -131,31 +143,31 @@ It is possible for a CSP to achieve the security and confidence comparable to in
+3. The CSP SHALL require all actions taken by the applicant during the enrollment and identity proofing process to be clearly visible to the remote operator. The operator SHALL direct the applicant, as required, to remove any doubt in the proofing process.
+4. The CSP SHALL require that all digital verification of evidence (e.g., via chip or wireless technologies) be performed by integrated scanners and sensors that are in the entire field of view of the camera and the remote, live operator.
 5. The CSP SHALL require operators to have undergone a training program to detect potential fraud and to properly perform a virtual in-process proofing session.
 6. A CSP MAY have an attendant participate in-person, at the same physical location as the applicant, for the entirety of the enrollment and identity proofing session.
-7. The CSP SHALL employ physical tamper detection and resistance features appropriate for the environment in which it is located. For example, a kiosk located in a restricted area or one where it is monitored by a trusted individual requires less tamper detection than one that is located in a semi-public area such as a retail store.
-8. The CSP SHALL ensure that all communications take place over a mutually authenticated encrypted session.
+7. The CSP SHALL employ physical tamper detection and resistance features appropriate for the environment in which it is located. For example, a kiosk located in a restricted area or one where it is monitored by a trusted individual requires less tamper detection than one that is located in a semi-public area such as the concourse of a shopping mall.
+8. The CSP SHALL ensure that all communications take place over a mutually-authenticated encrypted session.
 
 ### <a name="trustref"></a> 5.3.4. Trusted Referee Requirements
 
-The CSP MAY use trusted referees, such as notaries, legal guardians, medical professionals, conservators, persons with power of attorney, or some other form of certified/trained/approved individuals that can vouch for or act on behalf of the individual in accordance with applicable laws, regulations, or agency policy.  The CSP MAY allow an individual that has successfully completed identity proofing to act as a trusted referee for another individual.  The CSP MAY use a trusted referee for both remote and in-person processes.  
+The CSP MAY use trusted referees, such as notaries, legal guardians, medical professionals, conservators, persons with power of attorney, or some other form of trained and approved or certified individuals that can vouch for or act on behalf of the applicant in accordance with applicable laws, regulations, or agency policy.  The CSP MAY allow an applicant that has successfully completed identity proofing to act as a trusted referee for another applicant.  The CSP MAY use a trusted referee for both remote and in-person processes.  
```

Comment:
Persons with power of attorney, legal guardians, and conservators act on behalf of the applicant and don't undergo any certification/training/approval process and so can't be considered trusted referees in the same sense as an attorney or medical or legal professional. It also seems dubious to allow anyone that has been identity proofed to be trusted to assist the process - what's the basis for the CSP trusting them?